src/Http/Controllers/VoyagerSettingsController.php

```diff
@@ -82,9 +82,9 @@ public function update(Request $request)
         $settings = Voyager::model('Setting')->all();
 
         foreach ($settings as $setting) {
-            $content = $this->getContentBasedOnType($request, 'settings', (object) [
+            $content = $this->getContentBasedOnType($request, 'public/settings', (object) [
```

Comment:
Why the `public/` path if I may ask?

Reply:
To upload into `storage/app/public/settings`.

Reply:
Thanks, will test this today 👍

Reply:
Ok

Reply:
I think we need a bit more cleanup if this is going to work this way.  That `public` path gets passed in as `$slug`, which is no longer accurate if we're passing a full path.  `$slug` is only used as a prefix to the storage path, so it should still work correctly.  I'd like to know why it broke, though... why this fix is needed in the first place.

Assuming this fix is actually necessary, i propose undoing this so it remains `$slug` (for future use), then adding the `public` prefix to the file and image path in `Controller::getContentBasedOnType()`